tools/gyp/.github/workflows/node-gyp.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         with:
           repository: nodejs/node-gyp
           path: node-gyp
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
```

Comment:
This file comes from upstream https://github.com/nodejs/gyp-next and looks like it was already done upstream in https://github.com/nodejs/gyp-next/pull/90 — @nodejs/gyp do we need a gyp-next update?